app/components/user_list/index.tsx

```diff
@@ -41,29 +56,51 @@ const sectionKeyExtractor = (profile: UserProfile) => {
     return profile.username[0].toUpperCase();
 };
 
-export function createProfilesSections(profiles: UserProfile[]) {
-    const sections: {[key: string]: UserProfile[]} = {};
-    const sectionKeys: string[] = [];
-    for (const profile of profiles) {
-        const sectionKey = sectionKeyExtractor(profile);
-
-        if (!sections[sectionKey]) {
-            sections[sectionKey] = [];
-            sectionKeys.push(sectionKey);
-        }
+const sectionRoleKeyExtractor = (intl: IntlShape, cAdmin: boolean) => {
+    // Group items by channel admin or channel member
+    return intl.formatMessage(cAdmin ? messages.admins : messages.members);
```

Comment:
If this is just about the keys, do we need the message translated? Shouldn't this be a more programmatic value, and the translation done in a different place?

Reply:
https://github.com/mattermost/mattermost-mobile/pull/6771/commits/fe9581ba9efe9bf72b94eeae3a44d76c84e16cdf